pkgs/development/python-modules/wolframclient/default.nix

```
@@ -0,0 +1,39 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, pyzmq
+, numpy
+, pytz
+, requests
+, aiohttp
+, oauthlib
+}:
+
+buildPythonPackage rec {
+  pname = "wolframclient";
+  version = "1.1.2";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "0vhgmr9i6qa98qv8sn7ph35m2gayjyif6d2q0sjbfdgihsh0ly8f";
+  };
+
+  propagatedBuildInputs = [
+    pyzmq
+    numpy
+    pytz
+    requests
+    aiohttp
+    oauthlib
+  ];
+
+  # see https://github.com/WolframResearch/WolframClientForPython/issues/18
+  doCheck = false;
+
```

do you mind adding some import checks, this will make sure that the package is compatible with it's dependencies
```suggestion

  pythonImportsCheck = [
    "wolframclient"
    "wolframclient.cli"
    ...
  ];
```

---

Cool :) Had no idea these existed.